src/test/java/uk/gov/hmcts/reform/waworkflowapi/util/LaunchDarklyFeatureTogglerTest.java

```diff
@@ -22,26 +24,23 @@
 
     @Test
     void should_return_default_value_when_key_does_not_exist() {
-        String notExistingKey = "not-existing-key";
-
         when(ldClient.boolVariation(
-            notExistingKey,
+            FeatureFlag.WA_NON_IAC_WARNINGS.getKey(),
             null,
-            true)
-        ).thenReturn(true);
+            false)
+        ).thenReturn(false);
 
-        assertTrue(launchDarklyFeatureToggler.getValue(notExistingKey, true));
+        assertFalse(launchDarklyFeatureToggler.getValue(FeatureFlag.WA_NON_IAC_WARNINGS, false));
     }
 
```

Comment:
I believe this test was doing something else could we revert it looks like it has overriden the previous test